docs/userguide/strategies.md

```diff
@@ -1,21 +1,34 @@
 # Strategies
```

Comment:
Please add a note at the beginning of the user guide here, explaining the renaming and that the user guide is still up-to-date with respect to functionality and that the names and stuff will be adjusted in the near future.
SPecifically with 0.8. coming soon we should have an appropriate warning here.

Reply:
Will do as part of followup PR2 as described in the PR description